generics/middleware/checkAdminRole.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * name : checkAdminRole.js
+ * author : Mallanagouda R Biradar
+ * Date : 7-Aug-2025
+ * Description : checkAdminRole middleware.
+ */
+var messageUtil = require('./lib/messageUtil');
+var responseCode = require('../httpStatusCodes');
+const jwt = require('jsonwebtoken');
+var reqMsg = messageUtil.REQUEST;
+
+module.exports = async function (req, res, next) {
```

Comment:
@MallanagoudaB update with similar changes that we discussed in for other services